src/material-examples/radio-overview/radio-overview-example.css

```diff
@@ -1,3 +1,3 @@
 .mat-radio-button ~ .mat-radio-button {
-  padding-right: 16px;
+  padding-left: 16px;
```

Comment:
This would be even better as a `margin`, rather than `padding`. The padding can lead to accidental clicks.

Reply:
Based on my testing, the padding does not cause an expansion of the click target because the `<label>` element is the click target. Though, I agree that `margin` is probably more appropriate.